skimage/filters/_median.py

```diff
@@ -48,7 +48,7 @@ def median(
 
     Returns
     -------
-    out : 2-D array (same dtype as input image)
+    out : 2-D array, same dtype as input image
```

Comment:
Is there any consensus/convention on '2-D' vs '2D' and 'array' vs 'ndarray?'

If so, I'll update #7518 accordingly.